src/graphics.zig

```diff
@@ -1912,6 +1912,7 @@ pub const Image = struct {
 };
 
 pub const Fog = struct {
+	fogColor: Vec3f,
 	color: Vec3f,
```

Comment:
Could you maybe improve the names here?
fog.color and fog.fogColor convey the same meaning to me.